source/process/casestudy.rst

```diff
@@ -2,7 +2,17 @@
 Case Studies
 =====================
 
-This document outlines some of the procedures and materials in developing an effective case study for Mattermost software. This template can be used both to describe deployments of the open source Mattermost Team Edition or commercial deployments with Mattermost Enterprise Edition. We're delighted to hear everyone's story and to share it with the community. 
+This document outlines some of the procedures and materials in developing an effective case study for Mattermost software. This template can be used both to describe deployments of the open source Mattermost Team Edition or commercial deployments with Mattermost Enterprise Edition. We're delighted to hear everyone's story and to share it with the community.
+
+Guest Article Announcements for Mattermost Apps and Services
+----------------------------------------------------------------------------
+
```

Comment:
Update so blogger knows their process, e.g. what to deliver in an email, example: https://docs.mattermost.com/process/partner-programs.html